### PR TITLE
feat: Add breadcrumbs to the project details page gf-165

### PR DESCRIPTION
**Pull request by AnnKasian**
Done:

- Added breadcrumbs to the project details page

Preview:
image

### CODE REVIEW

File: apps/frontend/src/libs/components/breadcrumbs/styles.module.css [hunk 1]
```diff
@@ -4,6 +4,7 @@
 	align-items: center;
 	justify-content: flex-start;
 	padding: 0;
+	margin-bottom: 32px;
```
Comment (liza-veis) [inline apps/frontend/src/libs/components/breadcrumbs/styles.module.css:7]: Common components shouldn't have margins
```suggestion
```

File: apps/frontend/src/pages/project/project.tsx [hunk 2]
```diff
@@ -37,6 +37,12 @@ const Project = (): JSX.Element => {
 
 	return (
 		<PageLayout isLoading={isLoading}>
+			<Breadcrumbs
+				items={[
+					{ href: "/", label: "Projects" },
+					{ label: project?.name || "" },
```
Comment (liza-veis) [inline apps/frontend/src/pages/project/project.tsx:43]: We know that we will always have project here. Let's not add additional checks just for typescript
```suggestion
					{ label: (project.name as ProjectType) },
```

Comment (AnnKasian) [reply apps/frontend/src/pages/project/project.tsx:43]: maybe just ```
{project && (
					<Breadcrumbs
						items={[{ href: "/", label: "Projects" }, { label: project.name }]}
					/>
				)}
```?

File: apps/frontend/src/pages/project/project.tsx [hunk 3]
```diff
@@ -37,6 +37,14 @@ const Project = (): JSX.Element => {
 
 	return (
 		<PageLayout isLoading={isLoading}>
+			<div className={styles["breadcrumb-container"]}>
+				{project && (
```
Comment (liza-veis) [inline apps/frontend/src/pages/project/project.tsx:41]: should be a boolean value
```suggestion
				{hasProject && (
```

**Review by liza-veis — CHANGES_REQUESTED** [review]

File: apps/frontend/src/pages/project/project.tsx [hunk 4]
```diff
@@ -37,6 +37,14 @@ const Project = (): JSX.Element => {
...
+					<Breadcrumbs
+						items={[{ href: "/", label: "Projects" }, { label: project.name }]}
```
Comment (yevhenii-hladkov) [inline apps/frontend/src/pages/project/project.tsx:43]:
```suggestion
						items={[{ href: AppRoute.ROOT, label: "Projects" }, { label: project.name }]}
```

**Review by what1s1ove — APPROVED** [review]
lgtm after guys' comments

File: apps/frontend/src/pages/project/project.tsx [hunk 5]
```diff
@@ -31,12 +31,25 @@ const Project = (): JSX.Element => {
 
 	const isRejected = projectStatus === DataStatus.REJECTED;
 
+	const hasProject = project !== null;
```
Comment (liza-veis) [inline apps/frontend/src/pages/project/project.tsx:34]: to keep it consistent with other places
```suggestion
	const hasProject = Boolean(project);
```

Comment (AnnKasian) [reply apps/frontend/src/pages/project/project.tsx:34]: yeah i tried it first, but the problem that `'project' is possibly 'null'` appears again

**Review by liza-veis — APPROVED** [review]

**Pull request merged**